.circleci/config.yml

```diff
@@ -181,7 +181,7 @@ workflows:
             - unit_testing_php_74
             - unit_testing_php_80
             - js
-      - e2e_chrome_wp_previous_major:
```

Comment:
Modifications in this file will be reverted before merging. It's just to be able to test the changes on the nightly release.